plookup/.gitignore

```diff
@@ -0,0 +1,2 @@
+/target
```

Comment:
Not needed now that this is in the plonky repo, I think

Reply:
Yes you're right. Actually, I'll just remove this `.gitignore`, since `Cargo.toml` is also in the plonky repo.